addons/account/models/account_payment_term.py

```diff
@@ -31,23 +31,33 @@ def _check_lines(self):
             if len(lines) > 1:
                 raise ValidationError(_('A Payment Term should have only one line of type Balance.'))
 
-    def compute(self, value, date_ref=False, currency=None):
+    def compute(self, company_value, foreign_value, date_ref, currency):
+        """Get the distribution of this payment term.
+
+        :param company_value (float): the amount to pay in the company's currency
+        :param foreign_value (float): the amount to pay in the document's currency
+        :param date_ref (datetime.date): the reference date
+        :param currency (Model<res.currency>): the document's currency
```

Comment:
That's not the syntax of `:param:` info fields. The type is not parenthesized and comes between the tag name and the field name. And python typing annotations don't use `<>` for type parameters.